app/src/androidTest/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapperTest.kt

```diff
@@ -0,0 +1,44 @@
+package blockeq.com.stellarwallet.encryption
+
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+import blockeq.com.stellarwallet.utils.AccountUtils
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class KeyStoreWrapperTest {
+    val aliases = arrayOf("1234", "5678", "0987")
+    val mnemonic = arrayOf("abandon",
+            "ability",
+            "able",
+            "about",
+            "above",
+            "absent",
+            "absorb",
+            "abstract",
+            "absurd",
+            "abuse",
+            "access",
+            "accident")
+
+    @Test
+    fun clear_aliases() {
+        val context = InstrumentationRegistry.getTargetContext()
+        val keyStoreWrapper = KeyStoreWrapper(context)
+
+        aliases.forEach {
+            AccountUtils.getEncryptedMnemonicPhrase(context, mnemonic.joinToString(" "), null, it)
```

Comment:
it is a good test, but you should focus on the CUT (class under test)
`AccounUtils` should have his own test class.
Could you create the aliases using `KeystoreWrapper.createAndroidKeyStoreAsymmetricKey(pin)`

Reply:
the concept of mnemonic should not be known in this layer.

Reply:
Yeah that's true, I'm not writing good tests XD